### PR TITLE
Scrolling app window feature added

### PR DESCRIPTION
**Pull request by RahatAhmed171**
## Issue Referenced

<issue #7 > 

## Describe the Changes
Using Desktop Assistant, now a user can scroll any app window in three ways through the voice commands below:
<li> <b>Scroll (left/right/down/up)</b>:   It will only do simple scroll to the given direction </li>
<li> <b>Scroll to  (left/right/down/up)</b>:   It will only do scroll to the extreme of given direction </li>
<li> <b>Start scrolling (left/right/down/up) or stop scrolling </b>:   It will start or stop gradual scroll in the given direction </li>

### Change 1: <src/Assistant.py>
  To detect the scrolling commands and handling the condition when the query is `None` 
### Change 2: <src/Support.py>
  lmplementing the code to scroll
### Change 3: <src/VoiceInterface.py>
  Correcting a typo in the code

## Type of Change

- [x] New feature (Scrollable app windows)

## Additional Setup Requirements

To run the added feature you need to install:
`pip install pyautogui`
and
`pip install Pillow`

## Neccessary Conditions

- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] The changes do not involve any Private data leak/reveal.
- [x] Secrets, if any, are kept hidden via environment file, whose structure has been explained via documentation.

### CODE REVIEW

**Comment by vihar-s1** [comment]
The code seems to be fine but there are some formatting inconsistencies like redundant blank lines after if statement and more. Clear those issues and I will merge this PR. I am marking some of the inconsistencies for reference

File: src/Assistant.py [hunk 1]
```diff
@@ -1,16 +1,19 @@
 import re
 from datetime import datetime
-
+import threading
 import Support
 from VoiceInterface import VoiceInterface
 
 LISTENING_ERROR = "Say that again please..."
 
+
```
Comment (vihar-s1) [inline src/Assistant.py:9]: two line gap is enough...no need for having three blank lines

File: src/Assistant.py [hunk 2]
```diff
@@ -40,15 +43,17 @@ def listen_for_query(self) -> str:
     
     
     def execute_query(self, query: str) -> None:
+
```
Comment (vihar-s1) [inline src/Assistant.py:46]: redundant blank line

File: src/Assistant.py [hunk 3]
```diff
@@ -78,6 +83,34 @@ def execute_query(self, query: str) -> None:
...
+                scroll_thread,stop_scroll_event=Support.setup_scrolling()
+                if scroll_thread is None:  # Only start if not already scrolling
+                    Support.start_scrolling(direction)
+
+            elif 'stop scrolling' in query:
+                scroll_thread,stop_scroll_event=Support.setup_scrolling()
+                if scroll_thread is not None:
+                    Support.stop_scrolling()
+            
+
```
Comment (vihar-s1) [inline src/Assistant.py:99]: unnecessary gap between two `elif` blocks

File: src/Assistant.py (hunk 1 above)
Comment (RahatAhmed171) [reply src/Assistant.py]: Kept only two blank lines.

File: src/Assistant.py (hunk 2 above)
Comment (RahatAhmed171) [reply src/Assistant.py]: removed the lines

File: src/Assistant.py (hunk 3 above)
Comment (RahatAhmed171) [reply src/Assistant.py]: removed the blank lines. I also went the through the `Support.py` file then identified and removed some redundant/blank lines too.

**Review by vihar-s1 — APPROVED** [review]

**Comment by vihar-s1** [comment]
give @RahatAhmed171 100 points

**Comment by vihar-s1** [comment]
Ping me once on discord if points are not reflected on slop leaderboard

**Pull request merged**